CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## [Next]
+ - Refactored the particle system
+ - Cleaned up the code and remove buffer_utils
```

Comment:
I dont see this change reflected in the code?

Reply:
Same as the other comment